nodebuilder/share/window.go

```diff
@@ -0,0 +1,13 @@
+package share
+
+import (
+	"time"
+)
+
+// Window is a type alias for time.Duration used in nodebuilder
+// to provide sampling windows to their relevant components
+type Window time.Duration
```

Comment:
It seems it is used only to configure fx in Nodebuilder. Can we get rid of this alias in favor of standard time.Duration? or is it, mainly used to improve the readability?

Reply:
https://github.com/celestiaorg/celestia-node/commit/1e260c3179c88f0491885fe4dcee08034ff15047